atlasapi/network.py

```diff
@@ -144,15 +145,48 @@ def get(self, uri):
             logger.debug("Auth information = {} {}".format(self.user, self.password))
 
             return self.answer(r.status_code, r.json())
+        except Exception:
+            logger.warning('Request: {}'.format(r.request.__dict__))
+            logger.warning('Response: {}'.format(r.__dict__))
+            raise
+        finally:
+            if r:
+                r.connection.close()
+
+    def get_big(self, uri):
+        """Get request (max results)
+
+        This is a temporary fix until we re-factor pagination.
+
+        Args:
+            uri (str): URI
+
+        Returns:
+            Json: API response
+
+        Raises:
+            Exception: Network issue
+        """
+        r = None
+
+        try:
+            r = requests.get(uri,
+                             allow_redirects=True,
+                             params= {'itemsPerPage': 500},
```

Comment:
Nit: (since this is getting refactored anyways) pull from settings rather than leave hard coded?

Reply:
Great idea,  will make that change!

Reply:
Turns out there is already a value for this in `settings` , `Settings.itemsPerPage`.

Will use that, and will add mechanism to set these defaults through ENV variables.